style.css

```diff
@@ -367,3 +367,167 @@ h4 {
 
   color: #fff;
 }
+
+.form {
+  position: absolute;
+  top: 36px;
+  display: flex;
+}
+
+.form img {
+  position: absolute;
+  width: 110px;
+  height: 74.02px;
+  left: 0;
+  top: 4732px;
+}
+
+.form h1 {
+  position: absolute;
+  width: 290px;
+  height: 56px;
+  left: 16px;
+  top: 4794px;
+  font-family: "Playfair Display", "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+
+  /* identical to box height, or 50px */
+  color: #1a2236;
+}
+
+.form p {
+  position: absolute;
+  width: 338px;
+  height: 63px;
+  left: 16px;
+  top: 4895px;
+  font-family: Poppins, "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
+}
+
+.form .form-start {
+  top: 4960px;
+}
+
+.form .form-start .name {
+  /* Rectangle 31 */
+
+  box-sizing: border-box;
+  position: absolute;
+  width: 330px;
+  height: 52px;
+  left: 16px;
+  top: 4994px;
+  background: #fff;
+  border: 1px solid #dfdfe0;
+  border-radius: 4px;
+  padding-bottom: 20px;
+  padding-left: 10px;
+}
+
+.form .form-start .email {
+  /* Rectangle 31 */
+
+  box-sizing: border-box;
+  position: absolute;
+  width: 330px;
+  height: 52px;
+  left: 16px;
+  top: 5054px;
+  background: #fff;
+  border: 0.5px solid #dfdfe0;
+  border-radius: 4px;
+  padding-bottom: 20px;
+  padding-left: 10px;
+}
+
+.form .form-start textarea.message {
+  box-sizing: border-box;
+  position: absolute;
+  width: 330px;
+  height: 130px;
+  left: 16px;
+  top: 5115px;
+  background: #fff;
+  border: 1px solid #dfdfe0;
+  border-radius: 4px;
+  padding-top: 10px;
+  padding-bottom: 80px;
+  padding-left: 10px;
+  font-family: Arial, "Courier New", Courier, monospace;
+}
+
+.form .form-start button.start {
+  position: absolute;
+  width: 330px;
+  height: 50px;
+  left: 16px;
+  top: 5280px;
+  background: #1a2236;
+  font-family: Poppins, "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  border: 0;
+
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better to add the button interaction styling to page for the best user experience :+1: and also to replicate the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9&t=IemPH7jo9GPnILu3-0) :joy: 

you may kindly visit this[ link ](https://www.geeksforgeeks.org/css-buttons/)your more info on CSS buttons :+1: 

![image](https://user-images.githubusercontent.com/102757126/208114725-f1461eb9-a41f-4ebe-ad80-23cd293d96dd.png)